Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift

```diff
@@ -230,3 +227,60 @@ extension Bookmarks {
         .padding()
     }
 }
+
+/// Deprecated members
+public extension Bookmarks {
```

Comment:
Extensions cannot be documented, so there is no difference between this comment and an equivalent comment with only two slashes. Also, no period. Alternatively, you could do what I suggested, putting it inline with the declaration itself. I like that approach because extensions are often used to group together declarations for a particular purpose, usually related to a protocol and the protocol name goes after the type name in the extension declaration. So it fits with the mental model of looking at the extension declaration itself to see why it is there.

Reply:
Ah okay, I've moved it inline as suggested.

I wasn't sure if you were actually suggesting placing it there originally.